src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/dwpuploadresponse/DwpUploadResponseMidEventHandler.java

```diff
@@ -0,0 +1,69 @@
+package uk.gov.hmcts.reform.sscs.ccd.presubmit.dwpuploadresponse;
+
+
+import static java.util.Objects.requireNonNull;
+
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.sscs.ccd.callback.Callback;
+import uk.gov.hmcts.reform.sscs.ccd.callback.CallbackType;
+import uk.gov.hmcts.reform.sscs.ccd.callback.PreSubmitCallbackResponse;
+import uk.gov.hmcts.reform.sscs.ccd.domain.*;
+import uk.gov.hmcts.reform.sscs.ccd.presubmit.PreSubmitCallbackHandler;
+
+
+@Component
+@Slf4j
+public class DwpUploadResponseMidEventHandler implements PreSubmitCallbackHandler<SscsCaseData> {
+
+    public static final String APPENDIX_12_DOC_NOT_FOR_SSCS5_CONFIDENTIALITY = "An Appendix 12 document cannot be uploaded with Confidentiality documents";
+
+    @Autowired
```

Comment:
Don't think you really need this constructor? Up to you but it should work without.

Reply:
I suggest keep it, so that it can be reuse in DwpUploadResponseAboutToSubmitHandlerTest.java

Reply:
You get one for free anyway without declaring it. As an example see here: https://github.com/hmcts/sscs-tribunals-case-api/blob/master/src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/uploadwelshdocument/UploadWelshDocumentsAboutToStartHandlerTest.java#L44 

UploadWelshDocumentsAboutToStartHandler doesn't have a constructor. Anyway, up to you.